src/main/java/cholog/wiseshop/api/campaign/service/CampaignService.java

```diff
@@ -127,4 +131,25 @@ public List<MemberCampaignResponse> readMemberCampaign(Member member) {
                 return MemberCampaignResponse.of(campaign, products.getFirst());
             }).toList();
     }
+
+    public List<ReadCampaignResponse> searchByProductName(String keyword, int page, int size) {
+        keyword = keyword.trim();
+        Pageable pageable = PageRequest.of(page, size, Sort.by("id").ascending());
+        Page<Product> pages = productRepository.findByNameContaining(keyword, pageable);

```

Comment:
```suggestion
        Page<Product> pages = productRepository.findAllByNameContaining(keyword, pageable);
```
사소한건데 All 붙이기로 했었던걸로 기억하고 있어용